akka-http/src/main/scala/akka/http/scaladsl/server/RequestContext.scala

```diff
@@ -42,6 +44,7 @@ trait RequestContext {
    */
   def settings: RoutingSettings
 
+  implicit def parserSettings: ParserSettings
```

Comment:
does it have to be implicit?
please add newline after it and add a short scaladoc comment